locations/tempelwg.yml

```diff
@@ -0,0 +1,159 @@
+---
+location: tempelwg
+location_nice: U-Alt Tempelhof
+latitude: 52.465551
+longitude: 13.38598
+contacts:
+  - "@nick:matrix.riotcat.org"
+
+hosts:
+  - hostname: tempelwg-core
+    role: corerouter
+    model: "dlink_covr-x1860-a1"
+    wireless_profile: tempelwg
```

Comment:
missing mac-override. mac could be read with this command: https://github.com/freifunk-berlin/bbb-configs/blob/main/group_vars/model_dlink_covr_x1860_a1.yml#L12

Reply:
But we have no mac override fur eth1? However similar locations with the same device also only have eth0 mac override...

Reply:
on my router it looks like this:

![image](https://github.com/user-attachments/assets/2a0ce1d7-0726-4aaf-8a76-cb81b0691fe2)

My understanding is that both ports are connected to a switch (https://github.com/openwrt/openwrt/blob/d0a2b3f824c5e283caf31ceea49abf3022e7dc51/target/linux/ramips/dts/mt7621_dlink_covr-x1860-a1.dts#L191-L203) and therefore there is only eth0, correct me if I'm wrong.

The real question for me is why isn't the mac properly set. From my undestanding and looking at the DAP X1860 the dts file might just miss the part where the mac-address is read: https://github.com/openwrt/openwrt/blob/d0a2b3f824c5e283caf31ceea49abf3022e7dc51/target/linux/ramips/dts/mt7621_dlink_dap-x1860-a1.dts#L203-L204

It might be a good idea to actually just fix the board definitions for all devices where we currently use mac_overrides.

Reply:
Yeah you are right. But I also have an eth1 interface if you look into `ip l`. ^^

Reply:
Did you try without doing a mac override? For me the dts looks at first glance okay? What do you want to add to the dts? Maybe you are right but I need a closer look.